Games/types/Mafia/Game.js

```diff
@@ -65,7 +65,7 @@ module.exports = class MafiaGame extends Game {
           labels: ["hidden", "absolute", "uncontrollable"],
           game: this,
           run: function () {
-            this.target.kill("leave", this.actor);
+            this.target.kill("leave", this.actor, true);
```

Comment:
you want players to instantly die when they leave? this was specifically designed so that if someone ragequits when they're about the be voted off, the game is not affected.